elliott/elliottlib/bzutil.py

```diff
@@ -584,15 +584,34 @@ def get_flaw_bugs(self, bug_ids: List, strict: bool = True, verbose: bool = Fals
         raise NotImplementedError
 
 
+def get_jira_name_id_mapping() -> dict:
+    """
+    Assuming that no two JIRA fields have the same name
+    """
+    result_json = requests.get(constants.JIRA_API_FIELD).json()
```

Comment:
Would not mind a retry here for the web request.

Reply:
added in https://github.com/openshift-eng/art-tools/pull/499/commits/28f35844d0de48d65dfff4642e498da1d97ce488